src/conftest.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+from typing import Optional
+
+from lightning_utilities.core.imports import package_available
+
+if package_available("pytest") and package_available("doctest"):
```

Comment:
Can we make this check more restrictive so it only runs during doctests? Basically asking if this code also will run on `import torchmetrics`?

Reply:
no, it shall not as it is outside `torchmetrics` package, it is directly in `src/`